### DISCUSSION TITLE
Problem with login on TV to capture charts..

### DISCUSSION THREAD

**Discussion by dennisholo** [FabCx/TradingView-Webhook-Bot#85] [Q&A]
Hello can somebody please help to take a look at this. I am ussing costum indicators and need to login in tradin[]() 
![IMG_20220217_085116_443](https://user-images.githubusercontent.com/84317307/154429939-0241cc84-f539-496d-a31d-a7a4b0ce62bf.jpg)
gvieuw for this.. I am having problems with login..

**Comment by dennisholo** [comment]
https://replit.com/@trendoscope/Tradingview-Telegram-Bot?v=1

**Reply by dennisholo** [reply]
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"[name="password"]"}
  (Session info: chrome=91.0.4472.101)
Stacktrace:
#0 0x559e3ee05919 <unknown>

**Reply by fabston** [reply]
My code does not provide screenshot capturing so I’m afraid I’m not able to assist you.
